can/can_graph.py

```diff
@@ -0,0 +1,294 @@
+#!/usr/bin/env python3
```

Comment:
Can you modify `can/README.md` to explain how to run your examples?

In particular, the paths to the `python` directory seem to make it so you need to run the `can_graph.py` and `can_graph_gui_simple.py` examples while in the `can` directory, but you need to run the `can_graph_gui_responsive.py` example while in the `gui` directory.  If this could be made consistent, it would be nice.  But also, if it was documented in the README (with a command to copy paste) how to run each file, it would also help.

Reply:
Oh, I just noticed that this all moves in the third and fourth commits.  Please remove the intermediate states from the PR.  It's not really helpful to do things a certain way only to move and change them in the following commit.  It's normal to do many trial and errors while designing, but for the PR, just keep the final one.